databases/schemas/stored-battles.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE stored_battles (
+	roomid TEXT NOT NULL PRIMARY KEY, -- can store both the num and the formatid
```

Comment:
Is this preferable to having the num and formatid in separate columns?

Reply:
It's more convenient usage-wise. Is it a problem storage-wise?

Reply:
It looks like later in the code you do `const [, formatid] = roomid.split('-')` so it might be better to just store them in the right format to begin with

Reply:
Yeah, but I also use the direct roomid a few times. Suppose it's not a big deal either way.

Reply:
Oh, actually, direct roomid is maybe better, since that stores the "password" / existence of a password on the roomid as well.

Reply:
Being able to search for battles in a particular format without having to deal with `LIKE` would make things like [`battlesearch`](https://crates.io/crates/battlesearch) simpler, and I think that `(format, gameid, password)` would provide just as much functionality as battlesearch? Ultimately it's not very important either way.

Reply:
This won't work for battlesearch. We delete the saved battles as they're loaded from the DB.

Reply:
Oh, this is meant to coexist with terabytes of JSON battle logs, not supplant it. Then it doesn't matter to me at all.